src/lib/mina.ts

```diff
@@ -232,18 +233,25 @@ function LocalBlockchain({
         Math.ceil((new Date().valueOf() - startTime) / msPerSlot)
       );
     },
-    getAccount(publicKey: PublicKey): Account {
-      let ledgerAccount = ledger.getAccount(publicKey);
+    getAccount(
+      publicKey: PublicKey,
+      tokenId: Field = getDefaultTokenId()
+    ): Account {
+      let ledgerAccount = ledger.getAccount(publicKey, tokenId);
       if (ledgerAccount == undefined) {
         throw new Error(
-          `getAccount: Could not find account for public key ${publicKey.toBase58()}`
+          `getAccount: Could not find account for public key ${publicKey.toBase58()} with the tokenId ${Ledger.fieldToBase58(
```

Comment:
can you omit the `with the tokenId ...` in the case that the tokenId is the default one?

Reply:
Good call, fixed!